### PR TITLE
feat(logging): integrate ReadBoard diagnostics and support bundle

### PR DESCRIPTION
**Pull request by qiyi71w**
> [!IMPORTANT]
> Depends directly on #294 and transitively on #293, #289, #288, #286, and #284.
> This PR is reviewable now, but must not merge before the preceding logging stack.
>
> Because the stack crosses a fork boundary, GitHub cannot use the fork-only #294 branch as the upstream base. Until #294 lands, this comparison includes the preceding logging commits. After #294 merges, this branch will be rebased onto `upstream/main` so the diff collapses to the cross-repo ReadBoard integration, dual-process bundle changes, and UI refinement.
>
> The ReadBoard counterpart should merge before this PR is made ready.

## Summary

- Add the frozen ReadBoard logging launch and wire contract after the existing seven positional arguments.
- Migrate ReadBoard host lifecycle and protocol observations from standard streams to safe SLF4J summaries.
- Add desired/observed control for ReadBoard Diagnostics, Full Trace, and Capture, including timeout, disconnect, restart, and legacy states.
- Extend the host-owned diagnostic bundle with namespaced Lizzie and ReadBoard logs, current capture events, snapshots, session boundaries, manifest source status, and export sanitization.
- Present Host and ReadBoard as equal panes in Diagnostics and Logs, with explicit persistence, drop count, process session, toggle presentation, and capture confirmation.
- Refine control alignment, log-stream layout, actions, and localized labels without adding controls to the ReadBoard helper UI.

## Type Of Change

- [ ] Bug fix
- [x] Feature
- [ ] Packaging / release update
- [ ] Docs / translation
- [ ] Refactor

## Testing

- [x] Tested locally
- [x] Verified package contents if packaging changed
- [ ] Verified Fox sync flow if related
- [ ] Added screenshots if UI changed
- [ ] Updated README / docs if user-facing behavior changed
- [x] Noted affected platforms if the change is platform-specific
- [ ] Verified there is no unrelated formatting or line-ending churn
- [ ] Ran `python3 scripts/check_line_endings.py` (or equivalent `python` on Windows)

## Native Acceptance

- Original Ticket 08 run: 9 PASS / 1 FAIL / 1 BLOCKED / 0 NOT RUN.
- The failure was ReadBoard false-healthy persistence for an ACL-denied explicit log directory.
- The ReadBoard patch candidate passed native Case 7 revalidation: capability ready, persistence unavailable, determinate toggles, unchanged explicit path, no app log, and no LocalAppData fallback.
- Performance Case 10 remains BLOCKED because the repository has no reproducible logging-overhead comparison harness. This PR does not claim full Ticket 08 acceptance.

## Cross-repository Counterpart

- ReadBoard PR: https://github.com/qiyi71w/readboard/pull/73
- The new helper remains compatible with old hosts; the new host remains compatible with old helpers.

## Notes

Review the dependency banner before merging. After #294 lands, rebase this branch onto the new `upstream/main`, force-push with lease, rerun CI, and re-check the narrowed diff before marking the PR ready.

### CODE REVIEW

File: src/test/java/featurecat/lizzie/logging/DiagnosticBundleExporterTest.java [hunk 1]
```diff
@@ -0,0 +1,869 @@
...
+  }
+
+  @Test
+  void manifestRecordsHelperMissingAndUnreadableSourcesWithoutFailingZip() throws Exception {
+    LoggingRuntime runtime = start();
+    Path rb = readBoardRoot(runtime);
+    Files.createDirectories(rb);
+    Path unreadable = rb.resolve("app.log");
+    Files.writeString(unreadable, jsonl(Instant.now().toString(), "app", PROCESS_SESSION, "hidden", null));
+    Files.setPosixFilePermissions(unreadable, Set.of());
```
Comment (qiyi71w) [inline src/test/java/featurecat/lizzie/logging/DiagnosticBundleExporterTest.java]: **STD-01** `IN_SCOPE` — `Files.setPosixFilePermissions` is unsupported on the Windows unit job (HEAD `e9adc1f`, job 97066772250: `UnsupportedOperation` at this line). Ticket 08 is a Windows-native helper contract; this is the unreadable-source case the PR cites from native Case 7.

Acceptance: skip/assume POSIX here, or set up unreadable files with a Windows-safe API, so `windows-script-validation` runs this test to completion.

File: src/main/java/featurecat/lizzie/logging/DiagnosticBundleExporter.java [hunk 2]
```diff
@@ -0,0 +1,1095 @@
...
+          "readboard-capture",
+          sourceRecord(
+              true,
+              "omitted",
+              0,
+              0,
+              limits.captureCapBytes(),
+              NS_CAPTURE,
+              processAlias,
+              "helper-not-started",
```
Comment (qiyi71w) [inline src/main/java/featurecat/lizzie/logging/DiagnosticBundleExporter.java]: **SPEC-01** `IN_SCOPE` — when the helper snapshot is attached but `processSession` is null (starting, capability timeout, or `onDisconnect`), capture is always recorded as `helper-not-started`. `copyReadBoardTrace` already branches `attached ? "no-current-session" : "helper-not-started"` a few lines above.

After a normal ReadBoard stop the host still has an attached control with a cleared session, so a support ZIP would claim the helper never started.

Acceptance: use the same attached/session ternary as `copyReadBoardTrace`, and cover attached-without-session in `DiagnosticBundleExporterTest`.

**Review by qiyi71w — COMMENTED** [review]
Rank: P2

Pinned: https://github.com/wimi321/lizzieyzy-next/pull/297 @ `e9adc1f25cc23434fb99bf755a3a79b4178d47ad` (draft, `feat/diagnostics-logs-ui-refinement` → `main`). Author is this `gh` account, so COMMENT only. GitHub comparison still includes Tickets 01–06 until #294 merges; this review is bound to the Ticket 08-only range vs #294 (`42bdde3ce252`…`e9adc1f`).

## Standards

Ticket 08 is focused: protocol/control/snapshot types, host handshake, dual-process export, equal panes, l10n.

**STD-01** `IN_SCOPE` `open` — `manifestRecordsHelperMissingAndUnreadableSourcesWithoutFailingZip` calls POSIX `Files.setPosixFilePermissions` and throws `UnsupportedOperationException` on the Windows unit job.

## Spec

Spec source: this PR body. Bound to the body only.

Covered: named logging argv after the existing seven positionals; capture forced off at launch; Unix/relative log dirs omit the suffix; desired/observed control with timeout/disconnect/restart/legacy/degraded presentation; host-owned ZIP namespaces; current-session capture binding; tagged JSONL sanitization; equal Host/ReadBoard panes; capture confirmation; l10n.

**SPEC-01** `IN_SCOPE` `open` — attached helper with no process session is labeled `helper-not-started` in `copyCapture`; `copyReadBoardTrace` already uses `no-current-session` for that case.

## Checks

HEAD `e9adc1f25cc23434fb99bf755a3a79b4178d47ad`, run https://github.com/wimi321/lizzieyzy-next/actions/runs/32587814726

- `build` **fail** — https://github.com/wimi321/lizzieyzy-next/actions/runs/32587814726/job/97066772320 — `EngineManagerLifecycleReservationTest.secondaryExplicitRestartBoardFenceTimeoutRetiresLifecycleFailClosed` (`primary.hasExclusiveGtpWorkInProgress()` still true). Not in the Ticket 08 file set; #294 `build` was green. Treated as SPEC-02 deferred flake until reproduced.
- `windows-script-validation` **fail** — https://github.com/wimi321/lizzieyzy-next/actions/runs/32587814726/job/97066772250 — STD-01 POSIX exporter test, plus `updateEnginesConvergesBothReplacementEnginesBeforeReady` loadsgf wait (unchanged in this range).

## Deferred

- **STD-02** `DEFERRED` `open` — UI changed; CONTRIBUTING screenshot box unchecked.
- **STD-03** `DEFERRED` `open` — user-facing diagnostics/logs UI; README/docs box unchecked.
- **SPEC-02** `DEFERRED` `open` — EngineManager CI on files this Ticket 08 range does not touch.
- **SPEC-03** `DEFERRED` `open` — Ticket 08 Case 10 remains BLOCKED, as the PR states.

Not re-opened: Tickets 01–06 findings; missing ADR. Spec is the PR body.

IN_SCOPE: Standards 1, Spec 1; DEFERRED: Standards 2, Spec 2

File: src/main/java/featurecat/lizzie/logging/DiagnosticBundleExporter.java (hunk 2 above)
Comment (qiyi71w) [inline src/main/java/featurecat/lizzie/logging/DiagnosticBundleExporter.java]: **SPEC-01** `IN_SCOPE` — still open after `190e8339` (this file is not in the delta). `copyCapture` uses a single `helper-not-started` when `!attached || processSession == null`. `copyReadBoardTrace` already branches `attached ? "no-current-session" : "helper-not-started"`. `ReadBoardLoggingSnapshot.from` sets `attached=true` for any live control, and `processSession()` treats empty as null, so STARTING / capability-timeout-before-session exports mislabel capture.

Acceptance: same ternary as `copyReadBoardTrace`; add a test with an attached helper and empty `processSessionId` that expects `no-current-session` (detached remains `helper-not-started`).

**Review by qiyi71w — COMMENTED** [review]
Rank: P2

Pinned: https://github.com/wimi321/lizzieyzy-next/pull/297 @ `190e8339acad288c157d86e150451aa652239a96` (draft, `feat/diagnostics-logs-ui-refinement` → `main`). Re-review of `e9adc1f` → `190e8339` (`test: stabilize cross-platform CI fixtures`). Author is this `gh` account, so COMMENT only. Bound to Ticket 08; Tickets 01–06 not re-litigated except CI on the two fixture files in this delta.

## Standards

**STD-01** `IN_SCOPE` `resolved` — POSIX `setPosixFilePermissions` is now gated on `supportsFileAttributeView("posix")`. Windows no longer throws; unreadable coverage is POSIX-only, missing-source still asserted. Hosted Windows unit suite is green.

## Spec

Spec source: this PR body. Bound to Ticket 08 / ReadBoard diagnostics only.

**SPEC-01** `IN_SCOPE` `open` — `copyCapture` is unchanged. An attached helper with a null/empty process session is still recorded as `helper-not-started`; `copyReadBoardTrace` already distinguishes `no-current-session`.

**SPEC-02** `DEFERRED` `resolved` — EngineManager fixtures were retimed: board-fence release moved before the dual `loadsgf` wait (sequencing, not a longer timeout), plus `awaitReservationReleased(secondary)` after fence settlement. Hosted `build` and `windows-script-validation` are green on this HEAD.

## Checks

HEAD `190e8339`, run https://github.com/wimi321/lizzieyzy-next/actions/runs/32588757477 — success (`build`, `windows-script-validation`).

## Deferred

- **STD-02** `DEFERRED` `open` — UI changed; CONTRIBUTING screenshot box unchecked.
- **STD-03** `DEFERRED` `open` — user-facing diagnostics/logs UI; README/docs box unchecked.
- **SPEC-03** `DEFERRED` `open` — Ticket 08 Case 10 remains BLOCKED, as the PR states.

Not re-opened: Tickets 01–06 findings; missing ADR. Spec is the PR body.

IN_SCOPE open: Spec 1; DEFERRED open: Standards 2, Spec 1

File: src/test/java/featurecat/lizzie/logging/DiagnosticBundleExporterTest.java (hunk 1 above)
Comment (qiyi71w) [reply src/test/java/featurecat/lizzie/logging/DiagnosticBundleExporterTest.java]: Resolved in 190e8339. The POSIX permission setup is now gated on PosixFileAttributeView support, so the test remains meaningful on POSIX filesystems and no longer fails on Windows. The Windows verification job passes with that change.

File: src/main/java/featurecat/lizzie/logging/DiagnosticBundleExporter.java (hunk 2 above)
Comment (qiyi71w) [reply src/main/java/featurecat/lizzie/logging/DiagnosticBundleExporter.java]: Fixed in b490856a. Capture export now reports `no-current-session` when ReadBoard is attached but has no process session, while a detached helper still reports `helper-not-started`. Added `attachedHelperWithoutProcessSessionOmitsCaptureAsNoCurrentSession`; it passes on both Linux and native Windows.

Comment (qiyi71w) [reply src/main/java/featurecat/lizzie/logging/DiagnosticBundleExporter.java]: Fixed in b490856a. Capture export now reports `no-current-session` when ReadBoard is attached but has no process session, while a detached helper still reports `helper-not-started`. Added `attachedHelperWithoutProcessSessionOmitsCaptureAsNoCurrentSession`; it passes on both Linux and native Windows.

**Review by qiyi71w — COMMENTED** [review]
Rank: P2

Pinned: https://github.com/wimi321/lizzieyzy-next/pull/297 @ `b490856a026284fd2d39dc4e6b89819a37bc903b` (draft, `feat/diagnostics-logs-ui-refinement` → `main`). Re-review of `190e8339` → `b490856a` (`fix(logging): report missing current helper session`). Author is this `gh` account, so COMMENT only. Bound to Ticket 08; Tickets 01–06 not re-litigated.

## Standards

No new `IN_SCOPE` findings. STD-01 remains resolved (POSIX chmod still gated).

## Spec

Spec source: this PR body. Bound to Ticket 08 / ReadBoard diagnostics only.

**SPEC-01** `IN_SCOPE` `resolved` — `copyCapture` now uses the same attached/session ternary as `copyReadBoardTrace`:

```java
request.readBoardLogging().attached() ? "no-current-session" : "helper-not-started"
```

`DiagnosticBundleExporterTest` asserts detached capture is `helper-not-started` and attached-without-session is `no-current-session`.

## Checks

HEAD `b490856a`, run https://github.com/wimi321/lizzieyzy-next/actions/runs/32591271676 — success.

- `build` **pass** — https://github.com/wimi321/lizzieyzy-next/actions/runs/32591271676/job/97075391362
- `windows-script-validation` **pass** — https://github.com/wimi321/lizzieyzy-next/actions/runs/32591271676/job/97075391264

## Deferred

Unchanged; not re-opened.

- **STD-02** `DEFERRED` `open` — UI changed; CONTRIBUTING screenshot box unchecked.
- **STD-03** `DEFERRED` `open` — user-facing diagnostics/logs UI; README/docs box unchecked.
- **SPEC-03** `DEFERRED` `open` — Ticket 08 Case 10 remains BLOCKED, as the PR states.

Not re-opened: Tickets 01–06 findings; missing ADR. Spec is the PR body.

IN_SCOPE open: 0; DEFERRED open: Standards 2, Spec 1
